packages/tests/package.json

```diff
@@ -2,21 +2,34 @@
   "name": "tests",
   "private": true,
   "scripts": {
-    "build": "gulp --gulpfile ./shadycss/gulpfile.js",
-    "build:watch": "chokidar --initial --follow-symlinks 'shadycss/module/**/*.js' --ignore 'shadycss/module/generated/**/*' 'node_modules/@webcomponents/shadycss/**/*.js' -c 'npm run build'"
+    "build": "gulp --gulpfile ./shadycss/gulpfile.js && rollup --config rollup.config.js",
```

Comment:
Feel free to punt to a follow up PR, but we should consider using wireit here.

Maybe esbuild as well, unless rollup's output is part of the system under test

Reply:
Yeah, I think moving from lerna to wireit + npm workspaces would be a good idea for a later PR. And no, the choice of Rollup isn't critical for the tests package. It's also used to build the scoped custom element registry polyfill and I haven't looked at that one very closely but I doubt the choice matters there either. However, some of the polyfills use Closure for type checking and as a smoke test for compatibility with internal build infra, so I'd have to spend more time digging into that to know if / how much of what Closure does here could be replaced by esbuild.